### ISSUE TITLE
Record when a Q was marked as answered

### ISSUE THREAD

**Issue by jonhoo** [jonhoo/wewerewondering#5]
Currently, we just record _if_ it's marked as such. Recording the time would allow the list of answered questions to be ordered by _when_ they were answered, which is handy for writing up notes after the fact (as I recently discovered).

**Comment by ghassanachi** [comment]
I'd be happy to take point on this. 

I started working on the implementation on [ghassanchi:q_answer_time](https://github.com/ghassanachi/wewerewondering/tree/q_answer_time). 

For now I am reusing the 'answered' field and made it an Option<usize>, got the local backend working, but I still need to setup a AWS environment to make sure everything is working with Lambda/Dynamo. Let me know if you would prefer a different approach (ex: a separate "answered_at" field, since it could remove the need for a migration). 

Once I'm sure the backend is working on Local and AWS,  I'll start working on the unit tests/ and client side code. 

PS: Frontend currently works with the timestamps, thanks to the diabolical magic of JS and its dynamic types :p

**Comment by ghassanachi** [comment]
Also just wanted to double check that you would be okay with the "answered" sorting to happen client side. I'm not sure if there is an easy way of getting the sorting done server side given the way we are currently loading the questions from the folowing endpoints 
```rust
        .route("/api/event/:eid/questions", get(list::list))
        /* ..... */ 
        .route("/api/questions/:qids", get(questions::questions))
```

**Comment by jonhoo** [comment]
Yep, totally okay for the answered sort to happen client-side I think :+1: 

As for having a template for getting set up, yeah, that's probably a good idea! Although my hope is actually that we could enable people to test locally with _only_ DynamoDB set up. I don't think the API Gateway + Lambda stuff is all that relevant to most testing. And if it's _just_ DynamoDB, I don't know if we need a whole template for setup.

I didn't know what Postman was :p But yeah, that seems like it might be useful. Although I usually just run it locally and use the UX to trigger the requests and watch them in the network tab. I can see how being able to call the APIs directly might be handy in some cases though.
